Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
Oh sorry I mixed up our Docker images!!! Yes, for the "dev" Docker image it should be close to the devbox, we should stick to Ubuntu